example/pubspec.yaml

```diff
@@ -12,6 +12,10 @@ dependencies:
   flutter_bloc: ^0.21.0
   hydrated_bloc:
     path: ../
+  path_provider_fde:
+    git:
+      url: https://github.com/google/flutter-desktop-embedding
+      path: plugins/flutter_plugins/path_provider_fde
```

Comment:
You should use a specific hash here; an unpinned git dependency is very fragile, especially for a project like FDE which has no forward compatibility guarantees of any kind. (As written this would break when it moves out if the repository for instance)

Reply:
good point, updated 👍